app/views/tolk/locales/show.html.erb

```diff
@@ -12,6 +12,12 @@
   <%= render :partial => "tolk/searches/form", :locals => { :locale => @locale } %>
 </div>
 
+<%= render :partial => "tolk/locales/gtranslate_all" %>
+
+<div class="select-tag limit-select">
+  Show <%= select_tag :limit, options_for_select([30, 60, 100], selected: @limit), id: "limit_select" %> translations per page
+</div>
```

Comment:
Two questions:
* Is selected value passed upon saving current page?
* What if user is on second page and then changes this? Would probably mess up the place in which you were, yes? That's probably less critical however...

Reply:
I think it is a default behavior to take to the first page in such a situation as you no longer know on which page you are

Reply:
The update is redirecting to the referer, so if you changed the limit from the default, you would have it in the URL I think